src/core/state.go

```diff
@@ -1044,7 +1044,7 @@ func NewBuildState(config *Configuration) *BuildState {
 			"crc64":  fs.NewPathHasher(RepoRoot, config.Build.Xattrs, newCRC64, "crc64"),
 			"blake3": fs.NewPathHasher(RepoRoot, config.Build.Xattrs, newBlake3, "blake3"),
 		},
-		ProcessExecutor: process.New(sandboxTool),
+		ProcessExecutor: process.New(sandboxTool != ""),
```

Comment:
I think we may still want a way of honouring the sandbox tool if that's set. Maybe it should basically go through the old method if it is - but we make the default `""` from here on and deprecate that field in future.

Reply:
Yeah sounds like a plan